lms/static/sass/shared/_header.scss

```diff
@@ -377,6 +388,19 @@ header.global-new {
 
   .nav-global {
     @include float(left);
+    .tab-nav-item{
+      .tab-nav-link{
+        height: 43px;
+        font-size: em(16);
+        color: $gray;
+        &:hover{
```

Comment:
Yes, that's the one. And the tab underneath the "Alerts" section is what I have my styles mimicking